src/author/entities/author.entity.ts

```diff
@@ -0,0 +1,7 @@
+export class Author {
```

Comment:
It is described incorrectly, it needs a columns.  also not Author name it AuthorEntity